rust/tw_bitcoin/src/modules/transactions/ordinals.rs

```diff
@@ -89,7 +75,9 @@ fn create_envelope(mime: &[u8], data: &[u8], internal_key: PublicKey) -> Result<
     for chunk in data.chunks(520) {
         // Create data buffer.
         let mut data_buf = PushBytesBuf::new();
-        data_buf.extend_from_slice(chunk).map_err(|_| Error::Todo)?;
+        data_buf
+            .extend_from_slice(chunk)
+            .expect("chunk size is too large");
```

Comment:
This definitely can fail if an RPC server returns data that has more than 0x100000000 bytes